### PR TITLE
Update artifact version for successful build

### PR DESCRIPTION
**Pull request by ImNoahDev**
Update artifact version for successful build

### CODE REVIEW

File: iOS App Signer.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -302,7 +302,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				MACOSX_DEPLOYMENT_TARGET = 10.9;
+				MACOSX_DEPLOYMENT_TARGET = 14.5;
```
Comment (kambala-decapitator) [inline iOS App Signer.xcodeproj/project.pbxproj:305]: what's the reason of this change?

Comment (ImNoahDev) [reply iOS App Signer.xcodeproj/project.pbxproj:305]: 10.9 is an invalid deployment target and causes the build to fail. Plus the changes made previously to the certificate path only work on newer versions of macOS and will cause a breaking change on older versions.

Comment (ImNoahDev) [reply iOS App Signer.xcodeproj/project.pbxproj:305]: Changing the upload attract version is necessary to resolve the depreciation warning.

**Comment by ImNoahDev** [comment]
A working build of iOS App Signer for MacOS Sequoia can be found here: [ImNoahDev/ios-app-signer/releases/tag/1.15.0](https://github.com/ImNoahDev/ios-app-signer/releases/tag/1.15.0)

File: iOS App Signer.xcodeproj/project.pbxproj (hunk 1 above)
Comment (kambala-decapitator) [reply iOS App Signer.xcodeproj/project.pbxproj:305]: > 10.9 is an invalid deployment target and causes the build to fail.

it's not a reason to set it to such a high one. 10.13 would be a good choice.

> Plus the changes made previously to the certificate path only work on newer versions of macOS and will cause a breaking change on older versions.

this is not true, check the change: https://github.com/DanTheMan827/ios-app-signer/commit/9ba3a18723d9599549ac2741512899da1e831496

**Comment by ImNoahDev** [comment]
I have changed the build version to 10.13

File: iOS App Signer.xcodeproj/project.pbxproj (hunk 1 above)
Comment (ImNoahDev) [reply iOS App Signer.xcodeproj/project.pbxproj]: I have updated this in a new commit

File: README.md [hunk 2]
```diff
@@ -3,6 +3,11 @@ This is an app for OS X that can (re)sign apps and bundle them into ipa files th
 
 Supported input types are: ipa, deb, app, xcarchive
 
+Changes made in this fork
+------
+The upload artifact version and the MacOS development target versions have been changed in order to have a successful autobuild. 
+A pull request has been made to the original project but as of 2/11/24 it has not been merged and therefore this is the most up-to-date version of iOS App Signer with successful builds for MacOS 15.0+
+
```
Comment (kambala-decapitator) [inline README.md:10]: please drop this from the PR

**Comment by ImNoahDev** [comment]
Sorry about that, the PR should be ready now

File: README.md (hunk 2 above)
Comment (ImNoahDev) [reply README.md]: Sorry about that, the PR should be ready now
